source/adapters/CMakeLists.txt

```
@@ -27,3 +25,7 @@ endif()
 if(UR_BUILD_ADAPTER_CUDA)
 add_subdirectory(cuda)
 endif()
+
+if(UR_BUILD_ADAPTER_HIP)
+add_subdirectory(hip)
```

```suggestion
    add_subdirectory(hip)
```

I notice that the L0 and CUDA if statements above are also missing this indentation

---

fixed